sdk/storage/azdatalake/service/client.go

```diff
@@ -54,6 +54,11 @@ func NewClient(serviceURL string, cred azcore.TokenCredential, options *ClientOp
 	if options == nil {
 		options = &ClientOptions{}
 	}
+	perCallPolicies := []policy.Policy{shared.NewIncludeBlobResponsePolicy()}
```

Comment:
Is it possible to test the changes in the service/filesystem as well?

Reply:
yes, I added some tests that test that path but within the same directory/file test

Reply:
added tests within directory/file to test construction from each level